--- sphinx-copybutton-0.2.6/setup.py.orig	2019-12-03 03:40:46.000000000 +0100
+++ sphinx-copybutton-0.2.6/setup.py	2022-03-20 07:44:45.836657992 +0100
@@ -19,6 +19,6 @@ setup(
                                         '_static/copybutton.js',
                                         '_static/copy-button.svg']},
     include_package_data=True,
-    install_requires=["flit", "setuptools", "wheel", "sphinx"],
+    install_requires=["setuptools", "sphinx"],
     classifiers=["License :: OSI Approved :: MIT License"]
 )
